velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -127,6 +160,27 @@ class SubstraitVeloxPlanConverter {
       const ::substrait::Expression& substraitFilter,
       std::vector<::substrait::Expression_ScalarFunction>& scalarFunctions);
 
+  /// Helper function to extract the attributes required to create a ProjectNode
+  /// used for interpretting Substrait Emit.
+  template <typename RelMessage>
+  EmitInfo GetEmitInfo(const RelMessage& rel, const core::PlanNodePtr& node) {
+    const auto& emit = rel.common().emit();
+    int emitSize = emit.output_mapping_size();
+    std::vector<core::TypedExprPtr> emitExpressions(emitSize);
+    std::vector<std::string> emitProjectNames(emitSize);
+    EmitInfo emitInfo;
```

Comment:
This variable is not used until after the loop. Move it there.

Reply:
Sure I will update.